exe/sheap

```diff
@@ -3,15 +3,16 @@
 require "sheap"
 require "irb"
 
-heap = "asdfasdf"
-
 usage = <<~USAGE
-Usage: sheap
-       sheap HEAPDUMP
-       sheap BEFORE AFTER
+Usage: sheap                       # Start an IRB session with Sheap loaded
+       sheap HEAPDUMP              # Load a heap dump within an IRB session
+       sheap BEFORE AFTER [LATER]  # Compare heap dumps within an IRB session
 USAGE
 
 if ARGV.size == 0
+elsif ARGV.size == 1 && ["help", "--help"].include?(ARGV[0])
```

Comment:
I don't think we should support "help" (without `--`), as we don't have other commands.

```suggestion
elsif ARGV.size == 1 && ["help", "--help"].include?(ARGV[0])
```